pkg/runtest/run.go

```diff
@@ -65,7 +65,7 @@ func (ctx *Context) log(msg string, args ...interface{}) {
 	ctx.LogFunc(fmt.Sprintf(msg, args...))
 }
 
-func (ctx *Context) Run() error {
+func (ctx *Context) Run(tests string) error {
```

Comment:
This should go into the Context object. It's specifically added to not pass 10 arguments though all functions. Tests look like perfect match for Context.

Reply:
Fixed, PTAL